src/libs/elektra/kdb.c

```diff
@@ -1168,14 +1216,22 @@ int kdbSet (KDB * handle, KeySet * ks, Key * parentKey)
 
 error:
 	keySetName (parentKey, keyName (initialParent));
-	// GLOBAL: prerollback
+	// GLOBAL: prerollback [init]
+	// GLOBAL: prerollback [max once]
+	// GLOBAL: prerollback [foreach]
+	// GLOBAL: prerollback [deinit]
```

Comment:
again, a split in the loop within elektraSetRollback would be required (prerollback and rollback). The foreach must be within the loop of elektraSetRollback.